easybuild/easyconfigs/g/GRIDSS/GRIDSS-2.12.0-foss-2020b.eb

```diff
@@ -0,0 +1,47 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'BinariesTarball'
+
+name = 'GRIDSS'
+version = '2.12.0'
+
+homepage = 'https://github.com/PapenfussLab/gridss/wiki/GRIDSS-Documentation'
+description = """GRIDSS is a module software suite containing tools useful for
+the detection of genomic rearrangements. GRIDSS includes a genome-wide
+break-end assembler, as well as a structural variation caller for Illumina
+sequencing data. GRIDSS calls variants based on alignment-guided positional de
+Bruijn graph genome-wide break-end assembly, split read, and read pair
+evidence."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+github_account = 'PapenfussLab'
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5f00c918804a4e0cbe7dd8c61963058841f2104a01c58fc34bc19a0cc8c1a763']
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('R', '4.0.5'),
```

Comment:
@fizwit can you use version `4.0.4` here? that's the default for the 2020b generation